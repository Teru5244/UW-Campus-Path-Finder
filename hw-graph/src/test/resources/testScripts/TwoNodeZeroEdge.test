# Create a graph
CreateGraph g

# Add two nodes
AddNode g n1
AddNode g n2

# Print the nodes in the graph
ListNodes g