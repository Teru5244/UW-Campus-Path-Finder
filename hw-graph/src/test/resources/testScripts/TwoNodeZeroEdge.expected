# Create a graph
created graph g

# Add two nodes
added node n1 to g
added node n2 to g

# Print the nodes in the graph
g contains: n1 n2